install/ui/src/freeipa/ipa.js

```diff
@@ -76,13 +76,6 @@ var IPA = function () {
         processData: false
     };
 
-    /**
-     * i18n messages
-     * @deprecated
-     * @property {Object}
-     */
-    that.messages = {};
```

Comment:
The patch set is touching basically all usages of `IPA.messages` except the ones in tests. This makes it good to finish the original idea and remove `IPA.messages` completely.  It will allow to not required `ipa` module in `translation` module and thus break a circular dependency and also make the codebase bit more cleaner.

Reply:
I completely agree with you. Last "messages" have been removed.